dashboard/src/main/java/org/openedx/learn/presentation/LearnViewModel.kt

```diff
@@ -43,14 +50,24 @@ class LearnViewModel(
 
     init {
         viewModelScope.launch {
-            _uiState.collect { uiState ->
-                if (uiState.learnType == LearnType.COURSES) {
-                    logMyCoursesTabClickedEvent()
-                } else {
-                    logMyProgramsTabClickedEvent()
+            launch {
```

Comment:
Why we need launch inside launch?

Reply:
We can't collect two continuous flows within a single launch block because the latter one would become inaccessible.  
The choice was between this approach or two separate `viewModelScope` launch blocks.

Reply:
I prefer the second choose, but it's not very important

Reply:
I will update it as part of another PR. 👍🏻